front/src/form/bsvhu/Operation.tsx

```diff
@@ -92,62 +101,64 @@ export default function Operation() {
 
         <RedErrorMessage name="destination.reception.weight" />
       </div>
-
-      {values.destination?.type === BsvhuDestinationType.Demolisseur && (
+      {values.destination?.reception?.acceptationStatus !== "REFUSED" && (
         <>
-          <h4 className="form__section-heading">Identification</h4>
+          {values.destination?.type === BsvhuDestinationType.Demolisseur && (
+            <>
+              <h4 className="form__section-heading">Identification</h4>
+              <div className="form__row">
+                <label htmlFor="destination.reception.identification.numbers">
+                  Identification des numeros entrant des lots ou des VHU (livre
```

Comment:
C'était déjà là avant mais : 

```suggestion
                  Identification des numéros entrants de lots ou de véhicules hors d'usage (livre de police)
```